### PR TITLE
Spot check sections/offices fields added

### PR DESCRIPTION
**Pull request by reinbach**
[ch15365]

### CODE REVIEW

File: src/etools/applications/audit/serializers/engagement.py [hunk 1]
```diff
@@ -380,7 +384,7 @@ class Meta(EngagementSerializer.Meta):
             'internal_controls', 'findings', 'face_form_start_date', 'face_form_end_date',
             'amount_refunded', 'additional_supporting_documentation_provided',
             'justification_provided_and_accepted', 'write_off_required', 'pending_unsupported_amount',
-            'explanation_for_additional_information',
+            'explanation_for_additional_information', 'sections', 'offices',
```
Comment (reinbach) [inline src/etools/applications/audit/serializers/engagement.py]: I've added the `sections` to the serializer, but this field is not showing up in the serializer data? Does anyone have any idea why this is the case? I'm stumped!

**Comment by domdinicola** [comment]
@reinbach I guess you have to setup the permissions in the `update_audit_permissions` file
then run the command in order to update the permissions

File: src/etools/applications/audit/serializers/engagement.py (hunk 1 above)
Comment (domdinicola) [reply src/etools/applications/audit/serializers/engagement.py]: @reinbach I guess you have to setup the permissions in the update_audit_permissions file
then run the command in order to update the permissions

Comment (reinbach) [reply src/etools/applications/audit/serializers/engagement.py]: @domdinicola thanks, looks like that may be it

Comment (robertavram) [reply src/etools/applications/audit/serializers/engagement.py]: SeparatedReadWriteField magic!

Comment (domdinicola) [reply src/etools/applications/audit/serializers/engagement.py]: that's the future @robertavram !

File: src/etools/applications/audit/views.py [hunk 2]
```diff
@@ -444,6 +444,7 @@ class SpotCheckViewSet(EngagementManagementMixin, EngagementViewSet):
     queryset = SpotCheck.objects.all()
     serializer_class = SpotCheckSerializer
     csv_export_serializer = SpotCheckDetailCSVSerializer
+    serializer_action_classes = {}
```
Comment (domdinicola) [inline src/etools/applications/audit/views.py]: just curious.. why do we need this?

Comment (reinbach) [reply src/etools/applications/audit/views.py]: In an inherited View the `list` action is mapped to a specific serializer, which does not include the `offices`/`sections` fields. So this was to clear that mapping

**Comment by domdinicola** [comment]
in the card https://app.clubhouse.io/unicefetools/story/15365/spot-check-record-to-capture-respective-section-and-office I see in the comments that it would be tied only to One Office... did this change?

**Comment by reinbach** [comment]
> in the card https://app.clubhouse.io/unicefetools/story/15365/spot-check-record-to-capture-respective-section-and-office I see in the comments that it would be tied only to One Office... did this change?

In a subsequent comment @ntrncic updated that to be many to many for Offices as well

**Pull request merged**